lib/src/cmaf/ham/mapper/hls/mapHlsToHam.ts

```diff
@@ -1,63 +1,67 @@
-import { parseHlsManifest } from '../../../utils/hls/hlsParser.js';
-import { uuid } from '../../../../utils.js';
-import { addMetadataToHls } from '../../../utils/manifestUtils.js';
-import type {
+import {
 	AudioTrack,
+	Presentation,
 	Segment,
 	SelectionSet,
 	SwitchingSet,
 	TextTrack,
-	Track,
 	VideoTrack,
 } from '../../types/model';
 import type { Manifest, PlayList } from '../../types';
-
-function mapHlsToHam(manifest: Manifest) {
+import { addMetadataToHls } from '../../../utils/manifestUtils.js';
+import { parseHlsManifest } from '../../../utils/hls/hlsParser.js';
+import {
+	_formatSegments,
```

Comment:
remove the `_` in the method. We are using that for "private" functions that will not be exported.